### DISCUSSION TITLE
Transcript API seems to be down

### DISCUSSION THREAD

**Discussion by Vikrant-scoob** [deepgram/community#766] [General help] [answered]
Is there any push going on rn, need to test a few features. 
Will appreciate any input.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by SandraRodgers** [comment, marked as answer]
All systems are currently operational, which you can see on our status page: https://status.deepgram.com/

I also recommend testing requests on the playground to check that the API is up and running. https://playground.deepgram.com/

**Comment by jkroll-deepgram** [comment]
@Vikrant-scoob, checking in again - for further context, in recent days we have also been hearing similar customer reports from India and southeast Asia. Are you located in that region?

We have noticed some backbone internet issues stemming from that region of the world when trying to hit our datacenters, related to cross-region ISP network peering. We don't have any direct control over that, but have filed a ticket with upstream providers.

If you do notice the issue reoccurring, can you please run this test and share the results:

1. Current time when gathering this information (UTC timestamp)
1. IP address from a machine with connectivity issues (e.g. with `curl ifconfig.me` on Linux or `Invoke-WebRequest ifconfig.me` on Windows)
1. Traceroute using `traceroute api-alt.md1.deepgram.com`

We've heard some customers say that they've gotten around the issue by using a VPN.
